content/en/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions.md

```diff
@@ -408,7 +336,7 @@ and create it:
 kubectl create --validate=false -f my-crontab.yaml -o yaml
 ```
 
-you should get:
+you output is similar to:
```

Comment:
nit: This section may read better as a list instead of breaking up the commands and output.
```
1. Save the following YAML to my-crontab.yaml
2. Create the CronTab resource
3. Output
```

nit: `your output is similar to`